nncf/quantization/fake_quantize.py

```diff
@@ -177,25 +177,42 @@ def asymmetric_range(min_values: np.ndarray, max_values: np.ndarray,
 
 def calculate_quantizer_parameters(statistics: MinMaxTensorStatistic,
                                    quantizer_config: QuantizerConfig,
+                                   half_range: bool,
                                    quant_group: QuantizerGroup) -> FakeQuantizeParameters:
     """
     Calculates FakeQuantize layer attributes for weight/activation quantizer.
 
     :param statistics: Collected statistics for the quantized insertion.
     :param quantizer_config: Config of the quantization configuration.
+    :param half_range: If True effectively only a half of a quantizer range is used.
+        False - the full range is used.
     :param quantizer_group: Group of the quantizer.
     :return: Parameters of the FakeQuantize layer.
     """
     min_values = np.array(statistics.min_values).astype(np.float32)
     max_values = np.array(statistics.max_values).astype(np.float32)
 
+    num_bits = quantizer_config.num_bits
+    narrow_range = quantizer_config.mode == QuantizationMode.SYMMETRIC and quant_group == QuantizerGroup.WEIGHTS
```

Comment:
Since `cpu.json` contains `level_low` & `level_high` values, can we use them here?